src/components/EmojiPicker/EmojiPicker.js

```diff
@@ -176,7 +176,7 @@ const EmojiPicker = forwardRef((props, ref) => {
             onModalShow={focusEmojiSearchInput}
             onModalHide={onModalHide.current}
             hideModalContentWhileAnimating
-            shouldSetModalVisibility={false}
+            shouldSetModalVisibility={Browser.isMobile()}
```

Comment:
This also affects mChrome. Is it safe?
It would be good to add comment on platform specific changes like this.